src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3391,24 +3391,11 @@
      * <p>parses as:</p>
      *
      * <pre>
-     * +--VARIABLE_DEF
-     *     |
-     *     +--MODIFIERS
-     *         |
-     *         +--LITERAL_PUBLIC (public)
-     *     +--TYPE
-     *         |
-     *         +--IDENT (Collection)
-     *             |
-     *             +--TYPE_ARGUMENTS
-     *                 |
-     *                 +--GENERIC_START (&lt;)
-     *                 +--TYPE_ARGUMENT
-     *                     |
-     *                     +--WILDCARD_TYPE (?)
-     *                 +--GENERIC_END (&gt;)
-     *     +--IDENT (a)
-     *     +--SEMI (;)
+     * `--TYPE_ARGUMENTS -&gt; TYPE_ARGUMENTS
```

Comment:
please expand AST to start from `--VARIABLE_DEF -> VARIABLE_DEF [4:4]` to show more context and let user clearly match it to  example.

Reply:
Thanks for the comment! I did expand it, it is now starting from  `--VARIABLE_DEF -> VARIABLE_DEF`.